gcloud-java-contrib/gcloud-java-nio/pom.xml

```diff
@@ -77,24 +72,6 @@
       <artifactId>mockito-core</artifactId>
       <version>1.9.5</version>
     </dependency>
-    <dependency>
```

Comment:
That is fine. Then lets change the contrib readme (in this PR or next ones) to include the list of specific contrib modules and remove its artifact dependency description.